lib/message_builder.rb

```diff
@@ -192,20 +200,22 @@ def render(template_name)
   end
 
   def panda_presenter
-    prs_by_repo = dependapanda.group_by { |pr| pr[:repo] }
+    prs_by_repo = pull_requests.group_by { |pr| pr[:repo] }
 
     prs = prs_by_repo.map do |repo_name, prs_for_app|
       oldest_pr, newest_pr = prs_for_app.map { |pr| pr_date(pr) }.minmax
+      security_prs = prs_for_app.select { |pr| pr[:security_label] }
       {
         repo_name: repo_name,
         repo_url: "https://github.com/alphagov/#{repo_name}/pulls?q=is:pr+is:open+label:dependencies",
         pr_count: prs_for_app.count,
         oldest_pr: age_in_days(oldest_pr),
-        newest_pr: age_in_days(newest_pr)
```

Comment:
I know, I regretted not doing it as my first commit! I appreciate the commits are a bit messy, it's because I kept thinking I was done and then decided to change more stuff. I'll try to reshuffle things and also add some tests.